DTDL/rest-source.json

```diff
@@ -0,0 +1,18 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:io:tributech:source:rest;1",
+  "@type": "Interface",
+  "extends": "dtmi:io:tributech:source:base;1",
+  "displayName": "REST Source",
+  "contents": [
+    {
+      "@type": "Relationship",
+      "name": "Streams",
+      "minMultiplicity": 0,
+      "maxMultiplicity": 500,
+      "target": "dtmi:io:tributech:stream:restbase;1",
```

Comment:
i would suggest a dtmi like "dtmi:io:tributech:stream:rest:base;1", so you can use "dtmi:io:tributech:stream:rest:topic.. " etc